docs/usecase/usage-example.md

```diff
@@ -0,0 +1,62 @@
+# Use cases
+
+This document will introduce you the example of what Vald can do.
+Vald is the approximate nearest neighbor dense vector search engine. Vald supports the general usecase of similarity search engine, like faiss.
```

Comment:
I think we should describe unstructured data, it cannot search by Inverted index, and we shouldn't describe about faiss, please add another example.